core/validation/initial_check.py

```diff
@@ -22,6 +23,9 @@ def validate_before_transformation(workbook: dict[str, pd.DataFrame], reporting_
         workbook=workbook, reporting_round=reporting_round, place_names=place_names
     )
     file_validation_failures = pre_transformation_check(pre_transformation_details)
+    if reporting_round == 4:
+        if sign_off_failures := validate_sign_off(workbook):
+            file_validation_failures.extend(sign_off_failures)
```

Comment:
Bit nit picky but..

Could this be rewritten in this style:
```
if reporting_round == 4:
    sign_off_failures = validate_sign_off(workbook):
    file_validation_failures = [*file_validation_failures , *sign_off_failures]
```
for consistency with:
https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/045ab1caec8cb92a895d8db8b7d12f710f4d44eb/core/validation/__init__.py#L22-L24

Reply:
Yes, provided validate_sign_off() returns an empty list rather than None is the case where no error is raised. I've refactored the function so this is the case.